docs/root/source/data-models/block-model.rst

```diff
@@ -10,8 +10,8 @@ A block has the following structure:
       "block": {
         "timestamp": "<block-creation timestamp>",
         "transactions": ["<list of transactions>"],
-        "node_pubkey": "<public/verifying key of the node creating the block>",
-        "voters": ["<list of federation nodes verifying keys>"]
+        "node_pubkey": "<public key of the node creating the block>",
+        "voters": ["<list of federation nodes public keys>"]
```

Comment:
@sohkai There is a longer description of what "voters" means on the same page, below the outline of the JSON structure at the top.